src/org/loklak/objects/Timeline2.java

```diff
@@ -266,14 +266,26 @@ private Timeline2 addPost(Post post) {
         return this;
     }
 
+    public void mergePost(Timeline2 list) {
```

Comment:
I skipped this method as still UserEntry is not in application. I will have to set up Post object for this.